src/shogun/preprocessor/HomogeneousKernelMap.h

```diff
@@ -138,6 +129,10 @@ namespace shogun
 			 */
 			float64_t get_period() const;
 
+		protected:
+		    virtual SGMatrix<float64_t>
+		    apply_to_matrix(SGMatrix<float64_t> matrix) override;
```

Comment:
is there anywhere else a virtual function in this class? just make sure that in that case `override` is being used as otherwise clang is throwing a lot of warnings :)